src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -118,15 +120,21 @@ public boolean isUserRegistered(String username) {
     return false;
   }
 
+  public boolean isUserAdmin(String username) {
+    if (admin_usernames.contains(username)){
+      return true;
+    }
+    return false;
+  }
+
   /** returns the number of users. */
   public int getNumUsers(){
     return users.size();
   }
 
-  /** returns the user who sent the most messages */
-  public User mostActiveUser(){
-    User user = users.stream().max(Comparator.comparing(u -> u.getNumMessages())).get();
-    return user;
+  /** returns the newest user. */
+  public User newestUser(){
```

Comment:
What should we do if there are no users?

Reply:
If there are no users, the admin page will not be accessible because only admin users can access it, which means there would have to be at least one user - the admin before the methods could be called.